shlr/windbg/windbg.c

```diff
@@ -820,12 +827,19 @@ int windbg_sync(WindCtx *ctx) {
 
 	windbg_lock_enter (ctx);
 
+	ret = windbg_wait_packet (ctx, KD_PACKET_TYPE_UNUSED, NULL);
+	if (ret != KD_E_OK) {
+		ret = 0;
+		goto end;
+	}
```

Comment:
Pipe mode freezes here with:

`Opened pipe /tmp/windbg.pipe with fd 0x6`

Seeing that KD_PACKET_TYPE_UNUSED wasn't expected in the previous pipe implementation, this should have been seen as something that needs testing / limiting to net mode.

Reply:
Tried on windows 10 connecting to a windows 8.1 after manually 'fixing' the static symbol problem, hanged on this as well

Reply:
Seems to work now with Windows XP, can you confirm for 8.1 @GustavoLCR?